src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ApiService.scala

```diff
@@ -54,6 +54,15 @@ object ApiService extends LazyLogging with SprayJsonSupport with DefaultJsonProt
         complete(InternalServerError, AgoraException(e.getMessage, e.getCause, InternalServerError))
     }
   }
+
+  val rejectionHandler: RejectionHandler =
+    RejectionHandler.newBuilder().handle {
+      case MalformedRequestContentRejection(message, cause) => complete(BadRequest, message)
+    }.handle {
+      case UnacceptedResponseContentTypeRejection(supported) => complete(NotAcceptable,
+        "Resource representation is only available with these Content-Types:\n" + supported.map(_.format).mkString("\n"))
```

Comment:
This is how `spray` handled this case in `RejectionHandler`, but akka-http handles it slightly differently. Carrying the original approach over to maintain a friendly error response.